addons/mail/tests/test_mail_template.py

```diff
@@ -77,19 +77,26 @@ def test_mail_template_acl(self):
         self.assertFalse(self.user_employee.has_group('mail.group_mail_template_editor'))
         self.assertFalse(self.user_employee.has_group('base.group_sanitize_override'))
 
+        record = self.user_employee.partner_id
```

Comment:
test_mail_template_acl would probably benefit from being split into several sub tests, but hey that can be done "later" (not really something necessary for the freeze :D )